config/v1/types_authentication.go

```diff
@@ -195,74 +195,118 @@ const (
 )
 
 type OIDCProvider struct {
-	// name of the OIDC provider
+	// name configures the unique human-readable identifier
+	// associated with the identity provider.
+	// name is used to distinguish between multiple identity providers.
+	// name has no impact on token validation or authentication mechanics.
+	// name is required.
+	// name must not be an empty string ("").
```

Comment:
Instead of repeating `name ...` in the beginning of every sentence, would it make sense to make it a bit more natural, such as:
```suggestion
  // Name configures the unique human-readable identifier
	// associated with the identity provider.
	// Used to distinguish between multiple identity providers and 
	// has no impact on token validation or authentication mechanics.
	// It is required and must not be an empty string ("").
```

Also, we should probably match case in the doc comments.

This also seems to agree with our [API conventions](https://github.com/openshift/enhancements/blob/master/dev-guide/api-conventions.md#write-user-readable-documentation-in-godoc). This applies to various places in this diff.

Reply:
Yeah, looking back at this I went a little overboard with the repetition there 😅 - updating.

Reply:
>Also, we should probably match case in the doc comments.

It's API convention to match the serialized name, not the field name in the Go type

Reply:
For additional context on the serialized name matching, see: https://github.com/openshift/api/pull/2314#discussion_r2080147749